Flakey Test: ITElasticsearchHealthCheck.oneHealthy

```
[ERROR] zipkin2.server.internal.elasticsearch.ITElasticsearchHealthCheck.oneHealthy  Time elapsed: 0.078 s  <<< FAILURE!

org.opentest4j.AssertionFailedError: 

Expecting:

 <false>

to be equal to:

 <true>

but was not.

	at zipkin2.server.internal.elasticsearch.ITElasticsearchHealthCheck.oneHealthy(ITElasticsearchHealthCheck.java:106)
```

--- a/pom.xml
+++ b/pom.xml
@@ -50,9 +50,9 @@
 
     <!-- This allows you to test feature branches with jitpack -->
     <armeria.groupId>com.linecorp.armeria</armeria.groupId>
-    <armeria.version>1.1.0</armeria.version>
+    <armeria.version>1.2.0</armeria.version>
     <!-- Match Armeria version to avoid conflicts including running tests in the IDE -->
-    <netty.version>4.1.52.Final</netty.version>
+    <netty.version>4.1.53.Final</netty.version>
 
     <!-- It's easy for Jackson dependencies to get misaligned, so we manage it ourselves. -->
     <jackson.version>2.11.3</jackson.version>

--- a/zipkin-server/src/main/resources/simplelogger.properties
+++ b/zipkin-server/src/main/resources/simplelogger.properties
@@ -8,10 +8,7 @@ org.slf4j.simpleLogger.showShortLogName=true
 
 # this mirrors the logging configuration applied in zipkin-server-shared.yml , logging.level
 # This only includes Armeria as for example Kafka and Cassandra are not in the slim dist
-org.slf4j.simpleLogger.log.com.linecorp.armeria.client.HttpResponseDecoder=OFF
 org.slf4j.simpleLogger.log.com.linecorp.armeria.common.Flags=OFF
 org.slf4j.simpleLogger.log.com.linecorp.armeria.common.util.SystemInfo=WARN
-org.slf4j.simpleLogger.log.com.linecorp.armeria.internal.common.JavaVersionSpecific=OFF
-org.slf4j.simpleLogger.log.com.linecorp.armeria.server.docs.DocService=OFF
 org.slf4j.simpleLogger.log.com.linecorp.armeria.server.docs.DocStringExtractor=OFF
-org.slf4j.simpleLogger.log.com.linecorp.armeria.spring.ArmeriaAutoConfiguration=WARN
+org.slf4j.simpleLogger.log.com.linecorp.armeria.spring=WARN

--- a/zipkin-server/src/main/resources/zipkin-server-shared.yml
+++ b/zipkin-server/src/main/resources/zipkin-server-shared.yml
@@ -263,18 +263,12 @@ logging:
     com.datastax.oss.driver.internal.core.context.InternalDriverContext: 'WARN'
     # Use of native clocks in Cassandra is not insightful
     com.datastax.oss.driver.internal.core.time.Clock: 'WARN'
-    # Silence ResponseTimeoutException in the Armeria framework since we log it anyways in HTTP
-    # logging when enabled. https://github.com/line/armeria/issues/2000
-    com.linecorp.armeria.client.HttpResponseDecoder: 'OFF'
     com.linecorp.armeria.common.Flags: 'OFF'
-    com.linecorp.armeria.common.SerializationFormat: 'OFF'
     # Hostname logging is not Zipkin's duty
     com.linecorp.armeria.common.util.SystemInfo: 'WARN'
-    com.linecorp.armeria.internal.common.JavaVersionSpecific: 'OFF'
     com.linecorp.armeria.server.docs.DocStringExtractor: 'OFF'
-    com.linecorp.armeria.server.docs.DocService: 'OFF'
     # Redundant to logging already done in c.l.a.s.Server
-    com.linecorp.armeria.spring.ArmeriaAutoConfiguration: 'WARN'
+    com.linecorp.armeria.spring: 'WARN'
     # kafka is quite chatty so we switch everything off by default
     org.apache.kafka: 'OFF'
 #     # investigate /api/v2/dependencies

